lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -310,6 +333,59 @@ def get_driver_options(driver_options):
 
 class DockerNetworkManager(object):
 
+    def _get_minimal_versions(self):
+        # TODO: Move this and the same from docker_container.py to docker_common.py
+        self.option_minimal_versions = dict()
+        for option, data in self.client.module.argument_spec.items():
+            self.option_minimal_versions[option] = dict()
+        self.option_minimal_versions.update(dict(
+            scope=dict(docker_py_version='2.6.0', docker_api_version='1.29'),
```

Comment:
This one I cannot find in any docker API documentation (I checked up to 1.38). According to the docker-py code, it's supported from 1.30 on. Where did you got the information about the minimal API levels for these two options?

Reply:
https://docs.docker.com/engine/api/version-history/#v129-api-changes

"GET /networks/ now supports a scope filter to filter networks based on the network mode (swarm, global, or local)."

Reply:
You know what, I'll also increase this one to 1.30, just in case the docker-py folks know something we don't.

Reply:
The 1.29 API change might only refer to allowing to *filter* by scope, though I'd be surprised that the API only allows to filter when you can't *create* a network with a given scope.

It looks to me more and more like the docker API docs are kind of a mess ;-) Anyway, going to 1.30 is probably safe. If someone reports it also works with 1.29, we can still change it back.